### PR TITLE
Feature background image

### PR DESCRIPTION
**Pull request by rsnazario**
This is the first submit of the project 4 : Background images and gradient.

One question: the stickler keeps asking to change the indentation to 2 spaces instead of tabs. Is that really required?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <nav>
+                <ul>
+                    <li> <a href=""> <i class="fab fa-apple fa-lg"></i> </a> </li>
+                    <li> <a href=""> Store </a> </li>
+                    <li> <a href=""> Mac </a> </li>
+                    <li> <a href=""> iPod </a> </li>
+                    <li> <a href=""> iPhone </a> </li>
+                    <li> <a href=""> iPad </a> </li>
+                    <li> <a href=""> iTunes </a> </li>
+                    <li> <a href=""> Support </a> </li>
```
Comment (pelzolga123) [inline index.html]: `href` should not be empty. You can just add '#'.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                        <input type="text" > <i class="fas fa-search"></i>
+                    </li>
+                </ul>
+
+            </nav>
+
+            <div class="huge-container">    
+                
+                <div>    
+                    <a href=""><p class="what"></p></a>
```
Comment (pelzolga123) [inline index.html:34]: `href` should not be empty. You can just add '#'.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+                    <a href=""><p class="watch"></p></a>
+                </div>
+            </div>
+           
+            <div class="adv-containers">
+                <a href=""><p class="adv cell5s"></p></a>
+                <a href=""><p class="adv cell5c"></p></a>
+                <a href=""><p class="adv mac"></p></a>
+                <a href=""><p class="adv sust"></p></a>
```
Comment (pelzolga123) [inline index.html]: `href` should not be empty. You can just add '#'.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            </div>    
+
+        </div>
+
+        <footer>
+            <article>
+                <h6>Shop the <a href="">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href=""> Apple Retail Store,</a> or find a <a href=""> reseller.</a> </h6> <br>
```
Comment (pelzolga123) [inline index.html]: `href` should not be empty. You can just add '#'.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            
+            <ul>
+                <li> <a href=""> Site Map </a> </li>
+                <li> <a href=""> Hot News </a> </li>
+                <li> <a href=""> RSS Feedss </a> </li>
+                <li> <a href=""> Media Info </a> </li>
+                <li> <a href=""> Environment </a> </li>
+                <li> <a href=""> Job Opportunities </a> </li>
+                <li> <a href=""> Contact Us </a> </li>
+                <li> <a href=""> <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/global/elements/flags/16x16/usa.png" alt="Choose your country or region" height="16" width="16"> </a> </li>
```
Comment (pelzolga123) [inline index.html]: `href` should not be empty. You can just add '#'.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Pull request description should have info only about project. Stickler needs to be working correctly. Readme should be more descriptive. Bottom part requires some styling changes.

**Comment by pelzolga123** [comment]
## Hi Rafael!  
### Your project looks :+1: amazing!

 But there are things to change before this project can be approved:
* [ ] Your `.stickler.yml` file is present but not configured. You need to fix that. As for your question in pull request description: yes, you need to fix everything required, no matter how unimportant it seems. 
* [ ] Pull request description should only have information about changes in this project. For example:
  1. Added header with linear gradient background.
  2. Added bottom section with four images.
And so on. [Link to pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
* [ ] Readme file should be more descriptive. Please take a look at template [here](https://github.com/microverseinc/readme-template) to make it better. 
* [ ] Every `<a>` tag should have some link inside, to another pages or just '#'. 
* [ ] The bottom part styling is a little messed up, please make it look exactly like original. 
image

Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Hello @rsnazario, you have made a nice attempt!
There are a few things you need to look into before this project can be approved. Please take a look below:

##### Readme
Your project will benefit from a more descriptive readme file. It will make it look more professional and make your project look more interesting.
screenshot

##### Stickler
Stickler is still throwing errors as regards the code you've written, please look into it and make sure there are no errors before requesting another review.

##### Design
You made a nice attempt but please look very closely at the original page and ensure your design matches up closely. Look at the spacing between elements, the spacing in elements, the width of each element and make sure you can't tell much of a difference between the original site and your project.

## Summary
Please look into all that has been outlined above, improve the design, write a more descriptive readme file and correct stickler errors! When all this is done, request another review! Thanks :smile:

**Comment by rsnazario** [comment]
I am sorry about the README file, I was updating the one on Master Branch by mistake ><

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Nice job with the readme! @rsnazario 
Please take a look at the stickler errors next and lastly the layout. The current layout doesn't match the original page.

**Review by imhta — APPROVED** [review]
# Project approved 🎉 

### Hello  @rsnazario  , 👋 You have done a great job in this project 👏 




----
### Happy coding 😄 
----
**Pon Muthu Selvam N**
__nponmuthuselvam@gmail.com__

**Pull request merged**